From 9f06c60d451a1b92bda33eaca25f882634b9d5e7 Mon Sep 17 00:00:00 2001
From: Dimitri John Ledkov <dimitri.ledkov@surgut.co.uk>
Date: Wed, 4 Jun 2025 22:39:22 +0100
Subject: [PATCH 2/4] test_analyze_wheel_abi: add failing test case

Add a failing test case confiring that environment is not being
correctly reset.
---
 tests/integration/test_bundled_wheels.py | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/tests/integration/test_bundled_wheels.py b/tests/integration/test_bundled_wheels.py
index f7bc6296..0c1d2227 100644
--- a/tests/integration/test_bundled_wheels.py
+++ b/tests/integration/test_bundled_wheels.py
@@ -59,6 +59,12 @@
             frozenset([f"{HERE}/*"]),
             "LD_LIBRARY_PATH",
         ),
+        (
+            "cffi-1.5.0-cp27-none-linux_x86_64.whl",
+            {"libffi.so.5", "libpython2.7.so.1.0"},
+            frozenset([f"{HERE}/*"]),
+            None,
+        ),
         (
             "cffi-1.5.0-cp27-none-linux_x86_64.whl",
             {"libpython2.7.so.1.0"},
